src/modules/App.js

```diff
@@ -0,0 +1,121 @@
+/* eslint-disable */
```

Comment:
- [ ] I suggest you develop the practice of not disabling linters in your codebase, but rather research for solutions that will help you solve the linter issues because linters are set up to ensure you write clean and bug-free code. 😉 👍🏻

Reply:
Needs attention!